add community opensource mcp server OpenAPI AnyApi (#885)
modelcontextprotocol/servers: README.md

- **[OneNote](https://github.com/rajvirtual/MCP-Servers/tree/master/onenote)** - (by Rajesh Vijay) An MCP server that connects to Microsoft OneNote using the Microsoft Graph API. Reading notebooks, sections, and pages from OneNote,Creating new notebooks, sections, and pages in OneNote.
- **[OpenAI WebSearch MCP](https://github.com/ConechoAI/openai-websearch-mcp)** - This is a Python-based MCP server that provides OpenAI `web_search` build-in tool.
- **[OpenAPI](https://github.com/snaggle-ai/openapi-mcp-server)** - Interact with [OpenAPI](https://www.openapis.org/) APIs.

> - **[OpenAPI AnyApi](https://github.com/baryhuang/mcp-server-any-openapi)** - Interacting with large [OpenAPI](https://www.openapis.org/) docs using built-in semantic search for endpoints. It also allow customize mcp server prefix.

Reviewer: ```suggestion
- **[OpenAPI AnyApi](https://github.com/baryhuang/mcp-server-any-openapi)** - Interact with large [OpenAPI](https://www.openapis.org/) docs using built-in semantic search for endpoints. Allows for customizing the MCP server prefix.
```
